src/main/java/com/google/api/codegen/transformer/SurfaceTransformerContext.java

```diff
@@ -149,6 +149,21 @@ public MethodTransformerContext asMethodContext(Method method) {
     return methods;
   }
 
+  /** Returns a list of methods with samples, similar to getSupportedMethods,
```

Comment:
If you have a multi-line comment, I don't think the text is supposed to start on the first line